src/main/java/org/rutebanken/tiamat/exporter/StreamingPublicationDelivery.java

```diff
@@ -248,6 +295,113 @@ private void prepareStopPlaces(ExportParams exportParams, Set<Long> stopPlacePri
+                netexServiceFrame.setStopAssignments(stopAssignmentsInFrame_RelStructure);
+            }
+
+        }
+    }
+
+    private void covertStopPlaceToScheduledStopPoint(List<ScheduledStopPoint> scheduledStopPoints, List<JAXBElement<? extends StopAssignment_VersionStructure>> netexPassengerStopAssignment, org.rutebanken.tiamat.model.StopPlace stopPlace) {
+
+        // Add stop place
+        final String netexId = stopPlace.getNetexId();
+        String stopPlaceName = null;
+        if (stopPlace.getName() != null) {
+            stopPlaceName = stopPlace.getName().getValue();
+        }
+        final long version = stopPlace.getVersion();
+        var stopPlaceNetexId = netexId.split(":")[2];
+        var scheduledStopPointNetexId = "NSR:ScheduledStopPoint:S" + stopPlaceNetexId;
+
+        LocalDateTime validFrom = null;
+        LocalDateTime validTo = null;
+        if (stopPlace.getValidBetween() != null) {
+            if (stopPlace.getValidBetween().getFromDate() != null) {
+                validFrom = LocalDateTime.ofInstant(stopPlace.getValidBetween().getFromDate(), ZoneId.systemDefault());
+            }
+            if (stopPlace.getValidBetween().getToDate() != null) {
+                validTo = LocalDateTime.ofInstant(stopPlace.getValidBetween().getToDate(), ZoneId.systemDefault());
+            }
+        }
+
+
+        scheduledStopPoints.add(createNetexScheduledStopPoint(scheduledStopPointNetexId, stopPlaceName, version, validFrom, validTo));
+        netexPassengerStopAssignment.add(createPassengerStopAssignment(netexId, version, scheduledStopPointNetexId, passengerStopAssignmentOrder, validFrom, validTo, false));
+        passengerStopAssignmentOrder++;
+        // Add quays
+        final Set<Quay> quays = stopPlace.getQuays();
+        for (Quay quay : quays) {
+            final String netexId1 = quay.getNetexId().split(":")[2];
+            var scheduledStopPointNetexId2 = "NSR:ScheduledStopPoint:Q" + netexId1;
+            scheduledStopPoints.add(createNetexScheduledStopPoint(scheduledStopPointNetexId2, stopPlaceName, version, validFrom, validTo));
+            netexPassengerStopAssignment.add(createPassengerStopAssignment(quay.getNetexId(), quay.getVersion(), scheduledStopPointNetexId2, passengerStopAssignmentOrder, validFrom, validTo, true));
+
+            passengerStopAssignmentOrder++;
+        }
+
+    }
+
+    private JAXBElement<? extends StopAssignment_VersionStructure> createPassengerStopAssignment(String netexId, long version, String scheduledStopPointNetexId, int passengerStopAssignmentOrder, LocalDateTime validFrom, LocalDateTime validTo, boolean isQuay) {
+
+        final String passengerStopAssignmentId = scheduledStopPointNetexId.split(":")[2];
+
+        final PassengerStopAssignment passengerStopAssignment = new PassengerStopAssignment();
+        passengerStopAssignment.withId("NSR:PassengerStopAssignment:P" + passengerStopAssignmentId);
```

Comment:
Burde vel bruke netex.validPrefix - property-verdien i stedet for å hardkode NSR:?

Reply:
✅